x-pack/plugins/apm/public/components/app/RumDashboard/ClientMetrics/index.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+// @flow
+import * as React from 'react';
+import { EuiSpacer, EuiStat } from '@elastic/eui';
+import { useFetcher } from '../../../../hooks/useFetcher';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { BackEndLabel, FrontEndLabel, PageViewsLabel } from '../translations';
+
+export const formatBigValue = (val?: number | null): string => {
+  if (val && val >= 1000) {
+    return val / 1000 + 'k';
+  }
+  return val + '';
+};
+
+export const ClientMetrics = () => {
+  const { urlParams, uiFilters } = useUrlParams();
+
+  const { start, end } = urlParams;
+
+  const { data, status } = useFetcher(
+    (callApmApi) => {
+      if (start && end) {
+        return callApmApi({
+          pathname: '/api/apm/rum/client-metrics',
+          params: {
+            query: { start, end, uiFilters: JSON.stringify(uiFilters) },
+          },
+        });
+      }
+    },
+    [start, end, uiFilters]
+  );
+
+  return (
+    <>
+      <EuiSpacer size="l" />
+      <EuiStat
+        titleSize="m"
```

Comment:
```suggestion
        titleSize="s"
```
...for all of them.

Should look like this:
<img width="256" alt="Screenshot 2020-06-15 at 12 36 20" src="https://user-images.githubusercontent.com/20965076/84648147-da92b880-af04-11ea-812a-df7cb59207da.png">